src/qml/StatusbarItem.qml

```diff
@@ -9,6 +9,8 @@ Item {
     Layout.fillHeight: true
     Image {
         id: icon
+        width: root.height/2
```

Comment:
please do not refer to items that are not defined in this QML file :)

You can either add an "iconSize" property that you set when creating a StatusbarItem, or you can have a property that you then initialize with the component that creates StatusbarItem

Let me know if you need help :)